apis/ops/v1alpha1/mongodb_ops_helpers.go

```diff
@@ -48,3 +48,7 @@ func (m MongoDBOpsRequest) ResourcePlural() string {
 func (m MongoDBOpsRequest) ValidateSpecs() error {
 	return nil
 }
+
+func (m MongoDBOpsRequest) GetKey() string {
```

Comment:
"k8s.io/client-go/tools/cache"

Can we just use `cache.MetaNamespaceKeyFunc(obj)` ?